docs/search-configuration.md

```diff
@@ -0,0 +1,82 @@
+# Search Configuration
+
+Depending on your use case and requirements there are many ways you may wish to configure the search engine. The following built in configuration options aim to provide some flexibility out of the box however more advanced customisations may require some development work and deep knowledge of Elasticsearch.
+
+## Search Mode
+The mode setting determines whether or not the search will allow the use of advanced search syntax. The default mode is "simple" search. To enable advanced mode you would use the configuration below.
+
+```json
+{
+	"extra": {
+		"altis": {
+			"modules": {
+				"search": {
+					"mode": "advanced"
+				}
+			}
+		}
+	}
+}
+```
+
+By setting `mode` to "advanced" it is possible to write more complex search terms using operators:
+
+- Quoted strings will force an exact match
+- Brackets `(` and `)` denote precedence of terms
+- `AND` or a plus `+` character will match both the terms
+- `OR` or a pipe `|` character will match either of the terms
+- A word followed by `*` will match anything with that prefix
+- A word preceded by `-` will be excluded from the results
+
+For example the following search would return results with exact matches for "the law", or the words "judge" and "dred" with some degree of fuzziness and excluding the word "marvel".
+
+```
+"the law" OR (judge AND dred) -marvel
+```
+
+## Strict Search
+By default searching will match all the provided search terms with some degree of fuzziness to allow for misspellings and so on.
```

Comment:
I'm assuming `strict` is true by default, in which case why do we say "By default searching will match all the provided search terms with some degree of fuzziness to allow for misspellings and so on" which sounds like the opposite of strict?

Reply:
Yeah it is. Hmm, maybe we could flip it to “loose”. It’s essentially the difference between “or” and “and” so we could change it to “default_operator” instead

Reply:
I just thought a Boolean was simpler than a text value

Reply:
ah ok, I read this paragraph describing not-strict (fuzz), but you are saying "By default search is strict (using AND operators), but does allow for fuzzy word matching still" is that right?

Reply:
Yeah that's correct

Reply:
I updated this to remove the references to fuzzy matching as it's needlessly confusing. I think that's ok as behind the scenes behaviour.